core-feature-pack/src/main/resources/content/bin/standalone.bat

```diff
@@ -42,6 +42,7 @@ if not errorlevel == 1 (
 )
 setlocal DisableDelayedExpansion
 
+set outParam=
```

Comment:
[Original jira](https://issues.jboss.org/browse/WFCORE-3825) is just about domain scripts. Please update description of original jira, or move this fixes to different PR and different jira.